README.rst

```diff
@@ -11,9 +11,11 @@ Theme for Ontario ckan including:
 -------------------
 Directory Structure
 -------------------
+Three main directories compose this repository
```

Comment:
What do you propose doing with the `bin` directory? If it is to be kept, it should be added to this list.